src/config/constants.js

```diff
@@ -65,4 +65,19 @@ export const Context = {
   PLAYER: 'player',
   EXPLORER: 'explorer',
   ANALYZER: 'analyzer',
+  UNKNOWN: 'unknown',
 };
+
+// Used in ActionsByUserChart.js, maximum number of displayed users in the stacked bar
+export const ACTIONS_BY_USER_MAX_DISPLAYED_USERS = 10;
+
+// Used in ActionsByUserChart.js,
+export const ACTION_TYPES = [
+  'get',
+  'get_children',
+  'update',
+  'create',
+  'delete',
+  'copy',
+  'move',
+];
```

Comment:
Actually do you think it would be possible to get the list of types from the actions themselves instead of a hardcoded list? If we create new types, we will need to update manually this array.

Reply:
Then if there's a single user with 1 type, we cant have a complete set of actions, and we can't add 0 to those actions which are not used.

Reply:
I think instead you should get the whole list of types from all the actions (and not filtered by users). That way you have the complete list and can set 0 for users without this type.

Reply:
`const { actions, selectedUsers, allMembers } = useContext(DataContext);`
You mean the actions here? but if there's one one action, we can't have all the types

Reply:
For example, in the builder view, we only have one action
![image](https://user-images.githubusercontent.com/44131581/178703462-ccdbcea3-56e3-437e-a596-c8751bd33e61.png)

Reply:
Yeah, we could, but if there are many actions, we need to loop through all of them one by one to get a collection of different types?